docs/index.md

```diff
@@ -1 +1,71 @@
 # RedisAI Module
+
+RedisAI is a Redis module for serving tensors and executing deep learning models.
+
+## Quickstart
+
+1. [Docker](#docker)
+2. [Build](#building)
+3. [Start](#start)
+
+## Docker
+
+To quickly tryout RedisAI, launch an instance using docker:
+
+```sh
+docker run -p 6379:6379 -it --rm redisai/redisai
+```
+
+## Building
+This will checkout and build Redis and download the libraries for the backends (TensorFlow and PyTorch) for your platform.
+```
+mkdir deps
+DEPS_DIRECTORY=deps bash get_deps.sh
+
+cd deps
+git clone git://github.com/antirez/redis.git --branch 5.0
+cd redis
+make malloc=libc -j4
+cd ../..
+
+mkdir build
+cd build
+cmake -DDEPS_PATH=../deps/install ..
```

Comment:
-DDEPS_PATH=../deps-linux-gnu or -DDEPS_PATH=deps-darwin (use `ls` to see the actual directory name).